tutorial/stdlib2.po

```diff
@@ -183,24 +253,35 @@ msgid ""
 "`~logging.DEBUG`, :const:`~logging.INFO`, :const:`~logging.WARNING`, :const:"
 "`~logging.ERROR`, and :const:`~logging.CRITICAL`."
 msgstr ""
+"在預設情況，資訊和除錯訊息不會被顯示，其輸出會被發送到標準錯誤 (standard "
+"error)。其他輸出選項包括，將訊息轉發到電子郵件、資料報 (datagram)、網路插座 "
+"(socket) 或 HTTP 伺服器。新的過濾器可以根據訊息的優先順序，選擇不同的選路 "
+"(routing) 方式：\\ :const:`~logging.DEBUG`\\ ，\\ :const:`~logging.INFO`"
+"\\ ，\\ :const:`~logging.WARNING`\\ ，\\ :const:`~logging.ERROR`\\ ，及 :"
+"const:`~logging.CRITICAL`\\ 。"
 
 #: ../../tutorial/stdlib2.rst:240
 msgid ""
 "The logging system can be configured directly from Python or can be loaded "
 "from a user editable configuration file for customized logging without "
 "altering the application."
 msgstr ""
+"日誌記錄系統可以直接從 Python 配置，也可以從使用者可編輯的配置檔案載入，以進"
+"行客製化的日誌記錄而無須對應用程式做出變動。"
```

Comment:
align the terms used at [line 98](https://github.com/python/python-docs-zh-tw/pull/80/commits/02ce49f3ae82673341c6f571d3b33f875ec4d83b#diff-7e08c9470cbb581afb7cb1e79787f3553a95a24bb9826505a43914bc54122807R98)
```suggestion
"行客製化的日誌記錄，而無須對應用程式做出變更。"
```